### PR TITLE
Add dev config for AWS SQS

### PR DESCRIPTION
**Pull request by archferns**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/-LASB232)

Add config for Dev environment, and add temporary values for missing/unmapped mandatory MLRA attributes.

## Checklist

Before you ask people to review this PR:

- [x] Tests and linters should be passing
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/services/sqs/publish_laa_reference.rb [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 module Sqs
   class PublishLaaReference < ApplicationService
+    TEMPORARY_CJS_AREA_CODE = 16
+    TEMPORARY_CJS_USER = 'cpUser'
+    TEMPORARY_CJS_LOCATION = 'B16BG'
```
Comment (mpw5) [inline app/services/sqs/publish_laa_reference.rb]: I think maybe `TEMPORARY_USER` might be a better name here. I _think_ that the user details we're interested here are internal LAA users, so wouldn't really fall under CJS. No big deal but might just avoid confusion.

File: helm_deploy/laa-court-data-adaptor/values-dev.yaml [hunk 2]
```diff
@@ -14,6 +14,7 @@ nameOverride: ""
 fullnameOverride: ""
 
 common_platform_url: https://hmcts-common-platform-mock-api.apps.live-1.cloud-platform.service.justice.gov.uk
+sqs_region: eu-west-2
```
Comment (mpw5) [inline helm_deploy/laa-court-data-adaptor/values-dev.yaml]: Is it worth adding this to the `values-test`, `values-uat`, `values-stage` and `values-prod` files now too, just so they don't get forgotten?

File: app/services/sqs/publish_laa_reference.rb (hunk 1 above)
Comment (archferns) [reply app/services/sqs/publish_laa_reference.rb]: Good point, It is definitely clearer.

File: helm_deploy/laa-court-data-adaptor/values-dev.yaml (hunk 2 above)
Comment (archferns) [reply helm_deploy/laa-court-data-adaptor/values-dev.yaml]: Added

**Pull request merged**